lang/python/python3-readline/patches/010-link-with-system-libs.patch

```diff
@@ -0,0 +1,77 @@
+--- a/setup.py
```

Comment:
It's yours patch? 
- If yes, can you add commit header with commit description?
- If not, it comes from where and by whom?

Reply:
Yes, it is mine and willing to give it to you. And I rather not, it doesn't need one as it just allows compilation for openwrt, otherwise it will search for static archives from wrong places, mostly on host system and suddenly your musl is mixed with libc.

Reply:
Then it should be in the commit description.

Reply:
Provided although not useful.